dataprofiler/profilers/profile_builder.py

```diff
@@ -681,9 +681,9 @@ def __init__(
         self._min_true_samples: int = min_true_samples
         self.total_samples: int = 0
         self.times: dict[str, float] = defaultdict(float)
+        self._sampling_ratio: float = getattr(options, "sampling_ratio", 0.2)
 
         # TODO: allow set via options
-        self._sampling_ratio: float = 0.2
```

Comment:
setting `_sampling_ratio` through options if it is provided through the options -- otherwise it is set to the default that it was previously configured as -- `0.2`